Surface_mesh/examples/Surface_mesh/CMakeLists.txt

```diff
@@ -6,9 +6,14 @@ project( Surface_mesh_Examples )
 
 cmake_minimum_required(VERSION 2.8.11)
 
-if(NOT POLICY CMP0070 AND POLICY CMP0053)
+if(NOT POLICY CMP0070 AND  POLICY CMP0053)
   # Only set CMP0053 to OLD with CMake<3.10, otherwise there is a warning.
   cmake_policy(SET CMP0053 OLD)
+  elseif(POLICY CMP0070 AND  POLICY CMP0053)
+    cmake_policy(SET CMP0053 NEW)
+endif()
```

Comment:
That set of `if` is strange.
```cmake
if(POLICY CMP0053)
  if(NOT POLICY CMP0070)
    # Only set CMP0053 to OLD with CMake<3.10, otherwise there is a warning.
    cmake_policy(SET CMP0053 OLD)
  else()
    cmake_policy(SET CMP0053 NEW)
  endif()
endif()
```